action.yml

```diff
@@ -8,9 +8,36 @@ inputs:
     description: 'Additional metadata files that should be copied to the root of the deployment package. File patterns are supported, one pattern per line.'
     required: false
     default: ''
+  perform-nuget-verify:
+    description: 'A flag to indicate that the nuget signature verification step should be ran.'
+    default: 'true'
+  perform-assets-verify:
```

Comment:
Similar to the other PR, I would prefer to see this just work for anything passed into it, and not need these flags at all.